src/components/main/flight/CheckoutWithFlightScreen.js

```diff
@@ -142,254 +93,186 @@ export default class CheckoutWithFlightScreen extends React.Component {
-                  {
-                    color:
-                      this.state.treeNum == 0 ? COLORS.white : COLORS.lightblue,
-                  },
-                ]}
-                name="minus"
-              />
-            </TouchableOpacity>
-            {/*Tree counter*/}
-            <View style={[styles.treeCounter, { backgroundColor: color }]}>
-              <Text style={styles.treeCountText}>{treeNum}</Text>
+
+            {/* Number of Trees */}
+            <View style={styles.treeNumRow}>
+              <View style={styles.treeNumCol}>
+                <Image source={tree} style={styles.treeImg} />
+              </View>
+              <View style={styles.treeNumCol}>
+                <Text style={styles.numTrees}>{treeNum}</Text>
+              </View>
             </View>
-            {/*Add tree*/}
-            <TouchableOpacity
-              style={[styles.iterators, { backgroundColor: COLORS.lightgreen }]}
-              onPress={() => this.handleAdd()}>
-              <Feather
-                style={[styles.iteratorIcon, { color: COLORS.lightblue }]}
-                name="plus"
-              />
-            </TouchableOpacity>
-          </View>
-          <View style={styles.bottomText}>
-            {/*Years to neutralize carbon footprint*/}
-            <Text style={styles.midBlueText}>
-              {i18n.t("YEARS TO COMPENSATE")}
-            </Text>
-            {years != Infinity && (
-              <Text style={styles.bigBlueText}>{years}</Text>
-            )}
-            {years == Infinity && (
-              <Text style={styles.bigBlueText}>&#x2014;</Text>
-            )}
-          </View>
-          <Dash
-            style={styles.dashedLine}
-            dashColor={COLORS.lightgrey}
-            dashGap={0}
-          />
-          <View style={styles.receiptContainer}>
-            <View style={styles.textRow}>
-              {/*Total trees donated in transaction*/}
-              <Text style={styles.receiptTextLeft}>
-                {i18n.t("TOTAL TREES")}
+
+            {/* Add or Remove Trees */}
+            <View style={styles.addRemoveView}>
+              {/* Minus Icon */}
+              <View style={styles.iconView}>
+                <Ionicons
+                  style={styles.removeIcon}
+                  name="ios-remove-circle"
+                  onPress={() => this.handleRemove()}
+                />
+              </View>
+              {/* Add Icon */}
+              <View>
+                <Ionicons
+                  style={styles.addIcon}
+                  name="ios-add-circle"
+                  onPress={() => this.handleAdd()}
+                />
+              </View>
+            </View>
+
+            <View>
+              <Text style={styles.paragraph}>
+                {i18n.t("Emissions Statement")}{" "}
+                <Text style={{ fontWeight: "bold" }}>
+                  {years} {years > 1 && i18n.t("year") + "s"}
+                  {years == 1 && i18n.t("year")}
```

Comment:
years < 1???? Do something like MathRound(years * 12) "months".

and then when years == 0 do something like "a few days"

Reply:
this just to check to print, "year" or "years"

Reply:
yea but when it gets to less than a year it just says 0 (blank). we should have it calculate months if year<1